docs/en/operations/settings/settings.md

```diff
@@ -5682,3 +5682,25 @@ Default value: `0`.
 Enable `IF NOT EXISTS` for `CREATE` statement by default. If either this setting or `IF NOT EXISTS` is specified and a table with the provided name already exists, no exception will be thrown.
 
 Default value: `false`.
+
+## allow_suspicious_types_in_group_by {#allow_suspicious_types_in_group_by}
+
+Allows or restricts using [Variant](../../sql-reference/data-types/variant.md) and [Dynamic](../../sql-reference/data-types/dynamic.md) types in GROUP BY keys.
+
+Possible values:
+
+- 1 — Usage of `Variant` and `Dynamic` types is not restricted.
+- 0 — Usage of `Variant` and `Dynamic` types is restricted.
+
+Default value: 0.
+
+## allow_suspicious_types_in_group_by {#allow_suspicious_types_in_group_by}
+
+Allows or restricts using [Variant](../../sql-reference/data-types/variant.md) and [Dynamic](../../sql-reference/data-types/dynamic.md) types in GROUP BY keys.
+
+Possible values:
+
+- 1 — Usage of `Variant` and `Dynamic` types is not restricted.
+- 0 — Usage of `Variant` and `Dynamic` types is restricted.
+
+Default value: 0.
```

Comment:
It should be `order_by` and `ORDER BY`, and I also think listing the values in ascending order is better (just as we do for other settings)

```suggestion
## allow_suspicious_types_in_order_by {#allow_suspicious_types_in_order_by}

Allows or restricts using [Variant](../../sql-reference/data-types/variant.md) and [Dynamic](../../sql-reference/data-types/dynamic.md) types in ORDER BY keys.

Possible values:

- 0 — Usage of `Variant` and `Dynamic` types is restricted.
- 1 — Usage of `Variant` and `Dynamic` types is not restricted.

Default value: 0.
```